include/psa/crypto_platform.h

```diff
@@ -80,4 +80,10 @@ static inline int mbedtls_key_owner_id_equal( mbedtls_key_owner_id_t id1,
 
 #endif /* MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER */
 
+#if defined(MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG)
+typedef struct {
+    uint8_t opaque[32];
```

Comment:
isn't this very inflexible for something that is dependant on the underlying driver?

Reply:
Yes. This is a temporary place until we add support `crypto_drivers.h`. Once we finish implementing driver support, `mbedtls_psa_external_random_context_t` and `mbedtls_psa_external_get_random` will be generated from the driver description (although we may keep the ability to define the function manually until Mbed TLS 3.0 for backward compatibility). In the meantime, I don't know what to put here as a sample type. To keep the API simple, Mbed TLS allocates the context, so it needs to know the size.

Reply:
It's not ideal to have to define this type, but I don't see a better solution that can be done quickly.